src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,26 @@
 import React from 'react';
+import { Movie } from '../../react-app-env';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+interface Props {
+  movies: Movie[];
+}
+
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  return (
+    <div className="movies">
+      {movies.map((movie) => {
+        return (
+          <MovieCard
+            title={movie.title}
+            description={movie.description}
+            imgUrl={movie.imgUrl}
+            imdbUrl={movie.imdbUrl}
+            imdbId={movie.imdbId}
+          />
+        );
+      })}
```

Comment:
return is redundant here
```suggestion
      {movies.map((movie) => (
          <MovieCard
            title={movie.title}
            description={movie.description}
            imgUrl={movie.imgUrl}
            imdbUrl={movie.imdbUrl}
            imdbId={movie.imdbId}
          />
      ))}
```